keyboards/crkbd/keymaps/mcrown/keymap.c

```diff
@@ -0,0 +1,160 @@
+          /*|--------+--------+--------+--------+--------+----|     |--------+--------+--------+--------+--------+-----------|*/
+                            _____________MOD_LEFT_____________,      _____________MOD_RIGHT____________
+                        /*|-----------------------------------|     |----------------------------------|*/
+  ),
+
+  [_RAISE] = LAYOUT_wrapper( \
+          /*,-------------------------------------------------|-----|--------------------------------------------------------|.*/
+          _____________________RAISE_L1_______________________,      _____________________RAISE_R1_______________________,
+          /*|--------+--------+--------+--------+--------+----|     |--------+--------+--------+--------+--------+-----------|*/
+          _____________________RAISE_L2_______________________,      _____________________RAISE_R2_______________________,
+          /*|--------+--------+--------+--------+--------+----|     |--------+--------+--------+--------+--------+-----------|*/
+          _____________________RAISE_L3_______________________,      _____________________RAISE_R3_______________________,
+          /*|--------+--------+--------+--------+--------+----|     |--------+--------+--------+--------+--------+-----------|*/
+                            _____________MOD_LEFT_____________,      _____________MOD_RIGHT____________
+                        /*|-----------------------------------|     |----------------------------------|*/
+  ),
+
+  [_ADJUST] = LAYOUT_wrapper( \
+          /*,-------------------------------------------------|-----|--------------------------------------------------------|.*/
+          _____________________ADJUST_L1______________________,      _____________________ADJUST_R1______________________,
+          /*|--------+--------+--------+--------+--------+----|     |--------+--------+--------+--------+--------+-----------|*/
+          _____________________ADJUST_L2______________________,      _____________________ADJUST_R2______________________,
+          /*|--------+--------+--------+--------+--------+----|     |--------+--------+--------+--------+--------+-----------|*/
+          _____________________ADJUST_L3______________________,      _____________________ADJUST_R3______________________,
+          /*|--------+--------+--------+--------+--------+----|     |--------+--------+--------+--------+--------+-----------|*/
+                            _____________MOD_LEFT_____________,      _____________MOD_RIGHT____________
+                        /*|-----------------------------------|     |----------------------------------|*/
+  ),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record)
+{
+    bool user_records_press=TRUE;
+
+    if (record->event.pressed)
+    {
+        add_keylog(keycode);
+    }
+
+    switch (keycode)
+    {
+        case QWERTY:
```

Comment:
This can be removed, since it's the only default layer.

Reply:
Although it is the only default layer now, I'd rather to keep it like this because I'm planning to add more layer in a near feature :)